Unable to build multi-module project with fmp 3.5.34

### Description

When trying to build [this](https://github.com/snowdrop/spring-boot-configmap-booster/tree/sb-1.5.10) multi module project with fmp `3.5.34` I get the following error

```
[ERROR] Failed to execute goal io.fabric8:fabric8-maven-plugin:3.5.34:build (fmp) on project spring-boot-configmap-parent: Failed to execute the build: Unable to create the build archive: Failed to create assembly for docker image  (with mode 'dir'): Error creating assembly archive docker: You must set at least one file.. If you include the build artifact please ensure that you have built the artifact before with 'mvn package' (should be available in the target/ dir). Please see the documentation (section "Assembly") for more information.
```

I should not that when building other non multi-module projects, everything worked just fine.

In order to bypass the problem, I downgraded to fmp `3.5.33`, where everything worked as expected

### Info

* f-m-p version : 3.5.34
* Maven version (`mvn -v`) : 3.5.2
```

```
* Kubernetes / OpenShift setup and version : Openshift 3.7
* If it's a bug, how to reproduce : Run mvn clean verify -Popenshift,openshift-it on the sb-1.5.10 brach

### Notes
You can easily switch versions of fmp in the project by setting the `fabric8-maven-plugin.version` Maven property

--- a/doc/src/main/asciidoc/inc/goals/build/_fabric8-build.adoc
+++ b/doc/src/main/asciidoc/inc/goals/build/_fabric8-build.adoc
@@ -177,7 +177,7 @@ a| The build mode which can be
 | `docker.skip.build`
 
 | *skipBuildPom*
-| If set the build step will be skipped for modules of type `pom`. If not set, then by default projects of type `pom` will be skipped if there are no image configurations contained.
+| If set to `false` the build step will not be skipped for modules of type `pom`. By default the plugin is not executed for modules with `pom` packaging.
 | `docker.skip.build.pom`
 
 | *skipTag*

--- a/plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/BuildMojo.java
+++ b/plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/BuildMojo.java
@@ -105,7 +105,7 @@ public class BuildMojo extends io.fabric8.maven.docker.BuildMojo {
     @Parameter(property = "fabric8.resourceDir", defaultValue = "${basedir}/src/main/fabric8")
     private File resourceDir;
 
-    @Parameter(property = "fabric8.skip.build.pom")
+    @Parameter(property = "fabric8.skip.build.pom", defaultValue = "true")
     private Boolean skipBuildPom;
 
     /**
@@ -239,7 +239,8 @@ public class BuildMojo extends io.fabric8.maven.docker.BuildMojo {
             // If configured take the config option
             return skipBuildPom;
         }
-        // Not specified: Skip if no image with build configu configured, otherwise don't skip
+
+        // Not specified: Skip if no image with build configured, otherwise don't skip
         for (ImageConfiguration image : getResolvedImages()) {
             if (image.getBuildConfiguration() != null) {
                 return false;

